apidocs/custom-roles.md

```diff
@@ -0,0 +1,10 @@
+<blockquote>
+    <h3><span>📘</span>Available to enterprise customers</h3>
+    <p>Custom roles are available to customers on our enterprise plans. If you're interested in learning more about our enterprise plans, contact sales@launchdarkly.com.</p>
+</blockquote>
+
+Custom roles allow you to create flexible policies providing fine-grained access control to everything in LaunchDarkly-- from feature flags to goals, environments and teams. With custom roles, it's possible to enforce access policies that meet your exact workflow needs. 
+
+The custom roles API allows you to create, update and delete custom roles. You can also use the API to list all of your custom roles or get a custom role by ID.
+
+For more information about custom roles and the syntax for custom role policies, see the product documentation for  [Custom roles](https://docs.launchdarkly.com/docs/custom-roles).
```

Comment:
We should audit all of these links for correctness. This one is quite old -- at least it redirects to the right place but we may as well update it to the newer https://docs.launchdarkly.com/home/team/custom-roles.

Reply:
Thanks, yeah good shout. Reminds me I also need to fix the internal links.

Reply:
Audited external links and updated a couple - we do a great job of redirecting links, nothing 404'd 🎉